module-apps/tests/widgets/test-TimeFixedWidget.cpp

```diff
@@ -43,35 +29,31 @@ namespace
         EXPECT_TRUE(text.empty());
     }
 
-} // namespace
+    template <size_t N> auto initContainer(gui::HBox &box) -> gui::DigitsContainer<N>
+    {
+        auto container = gui::DigitsContainer<N>{};
+        for (auto &digit : container.digits) {
 
-TEST_F(TimeFixedWidgetTest, initilization)
-{
-    EXPECT_NE(widget, nullptr);
-}
+            digit = new gui::Label(&box, 0, 0, 0, 0);
+        }
+        return container;
+    }
 
-TEST_F(TimeFixedWidgetTest, minusNotVisable)
-{
-    widget->setFontAndDimensions(style::window::font::verybig);
-    EXPECT_EQ(widget->getLeftBox().minus->getText(), "");
-}
+} // namespace
 
-TEST(TimeFixedWidgetNonFixtureTest, minusVisable)
+TEST(TimeFixedWidgetTest, initilization)
```

Comment:
I'd remove this test case. Additionally, I'd rename the whole `test-TimeFixexWidget` to `test-DigitsContainer`.